### DISCUSSION TITLE
Woche 6 - Erste Demo

### DISCUSSION THREAD

**Discussion by lukashert** [Chrissi-Ruege/HealthKeeper#9] [General]
Hallo zusammen!

In der letzten Woche haben wir an einer ersten Demo gearbeitet. Da unser Fokus zunächst auf Arbeiten im Backend lagen, gibt es noch nicht allzu viele visuelle Ergebnisse.

### Statistik Seite
Auf der Statistik Seite können später Daten eingetragen und eingesehen werden. Das Frontend ist noch Work in Progress und wird sich in den nächsten Wochen noch drastisch ändern.

<img width="959" alt="frontend" src="https://github.com/Chrissi-Ruege/HealthKeeper/assets/20227840/fee1448f-6b81-4fac-bd48-e8343b7adcfd">

**Comment by Puggingtons** [comment]
Euer Frontend sieht doch schon mal nice aus :) Welche Arbeiten habt ihr denn im Backend bereits gemacht?

**Comment by DossohMael** [comment]
Gute Arbeit soweit, ich bin gespannt was noch daraus wird!
